src/fmu/sumo/sim2sumo/common.py

```diff
@@ -385,7 +392,12 @@ def add(self, file):
 
     def _upload(self):
         self._logger.debug("%s files to upload", len(self._objects))
-        nodisk_upload(self._objects, self._parentid, connection=self._conn)
+        nodisk_upload(
```

Comment:
Is this only used internally in sim2sumo?

Reply:
Kind off 🤷